system/security/securesocket/securesocket.cpp

```diff
@@ -1765,3 +2020,93 @@ ISmartSocketFactory *createSecureSmartSocketFactory(const char *_socklist, bool
 {
     return new CSecureSmartSocketFactory(_socklist, _retry, _retryInterval, _dnsInterval);
 }
+
+class CSingletonSecureSocketConnection: public CSingletonSocketConnection
+{
+public:
+    Owned<ISecureSocketContext> secureContextClient;
+    Owned<ISecureSocketContext> secureContextServer;
+    int tlsLogLevel;
+
+    CSingletonSecureSocketConnection(SocketEndpoint &_ep)
+    {
+        ep = _ep;
+        state = Snone;
+        cancelling = false;
+        secureContextClient.setown(createSecureSocketContextSecret("local", ClientSocket));
+        secureContextServer.setown(createSecureSocketContextSecretSrv("local"));
+#ifdef _CONTAINERIZED
+        tlsLogLevel = getComponentConfigSP()->getPropInt("logging/@detail", SSLogMin);
+        if (tlsLogLevel >= ExtraneousMsgThreshold) // or InfoMsgThreshold ?
+            tlsLogLevel = SSLogMax;
```

Comment:
most components have a default detail level set to 100 at the moment.
So SSLogMax will be the norm. , is that going to cause excessive tracing?